File: string/crawler-log-folder.cpp
class Solution {
public:
    int minOperations(vector<string>& logs) {
        int count =0;
        for (auto i : logs){
            if (i == "../"){
                count = max(0,count-1);
            }
            else if (i !="./"){
                count ++;
            }
        }
        return count;
    }
};

File: string/crawler-log-folder.py
class Solution:
    def minOperations(self, logs: List[str]) -> int:
        count = 0 
        for i in logs:
            if i == "../":
                if count >0:
                    count -=1
                else:
                    count=0
            elif i != "./":
                count+=1
        return count if count>=0 else 0
